docs/source/format/Columnar.rst

```diff
@@ -350,6 +352,51 @@ will be represented as follows: ::
     |----------------|----------------------|
     | joemark        | unspecified          |
 
+Variable-size Binary View Layout
+--------------------------------
+
+Each value in this layout consists of 0 or more bytes. These characters'
+locations are indicated using a **views** buffer, which may point to one
+of potentially several **data** buffers or may contain the characters
+inline.
+
+The views buffer contains `length` view structures with the following layout:
```

Comment:
The endianness of this data structure wasn't immediately apparent to me, I interpreted the view as being a single 128-bit integer with the native endianness. I believe this is consistent with intervals

Reply:
Not really. Each packed struct's field should have its endianness handled independently.
For example, short strings would be:
* in little-endian mode:
  * bytes 0-3 (little-endian int32)
  * data: bytes 4-15 (no endianness)
* in big-endian mode:
  * bytes 0-3 (big-endian int32)
  * data: bytes 4-15 (no endianness)